### DISCUSSION TITLE
Voti

### DISCUSSION THREAD

**Discussion by luigirotondi2009** [iisgiua/giuaschool#680] [Richieste di aiuto]
Se io voglio modificare le valutazioni esempio i = impreparato g = giustifica come posso farlo? C'è un metodo?

**Comment by trinko** [comment]
Il campo **GIUDIZIO** presente sotto ogni voto è pensato anche per questo utilizzo.

Altrimenti devi modificare il codice Twig dei template, inserire un po' di Javascript e forse (a seconda di cosa si vuole ottenere) modificare anche qualche file PHP.
